aten/src/ATen/native/native_functions.yaml

```diff
@@ -1885,6 +1885,14 @@
     CUDA: cudnn_batch_norm_backward
   autogen: cudnn_batch_norm_backward.out
 
+- func: cudnn_rms_norm(Tensor input, SymInt[] normalized_shape, Tensor? weight, float? epsilon=None) -> (Tensor, Tensor)
```

Comment:
New public functions like this is definitely a big nono

Why not update the existing function?

Reply:
We can make it "private" but the issue is that the current `rms_norm` is an unfused `CompositeImplicitAutograd` one, so the real way to add it would be as the `CUDA:` dispatch, but IMO that seems a bit heavy-handed to move every one immediately. This seems like the simplest way to keep it "opt-in" for now.